app/Lib/Repository/GitHub.php

```diff
@@ -65,35 +120,36 @@ public function __construct($token, $owner, $repo)
      *
      * @see https://developer.github.com/v3/repos/statuses/ for property descriptions
      *
-     * @param ClientInterface $client
      * @param array $options
-     * @return \GuzzleHttp\Promise\PromiseInterface
+     * @return array
```

Comment:
Is this method supposed to return something? There is no final `return` statement.